wagmi.config.ts

```diff
@@ -36,7 +41,7 @@ for (const key in rawJson) {
     contracts.push({
       name: contractName,
       address: rawJson[key][contractName],
-      url: `https://raw.githubusercontent.com/CamposBruno/accelerator-defi-EIP/fix/deployment/data/abis/${contractName}.json`,
+      url: `${env.SYNC_CONTRACTS_ABI_PATH}/${contractName}.json`,
```

Comment:
would the script fail gracefully with empty env var? like
`      url: `$undefined/${contractName}.json`,`

Reply:
I don't think we have any mandatory "env checks if variable present" yet